### PR TITLE
Fix #95

### LINKED ISSUE #95: Minor: formatting + small issues
Several small tasks:
1. **Fix links in Customize runtime pages**
* [ ] fix the extra line at `zCLI` (check all runtime services)
* [ ] fix the links (check all runtime services for the problem below)
  * I prefer the links to appear blue, so that I know its clickable. If you use ``<a href="/go/how-to/build-pipeline#preparecommands-1">`run.prepareCommands`</a>``, the link will appear grey. This `` `<a href="/go/how-to/build-pipeline#preparecommands-1">run.prepareCommands</a>` `` will break the link. Use `<code>` for proper formatting.
 Image
2. **How to invalidate runtime cache:**
* [ ] Find all occurrences of `` `yyy` `` and replace with "To invalidate the Zerops runtime cache go to your service detail in Zerops GUI, choose **Service dashboard & runtime containers** from the left menu and click on the **Open pipeline detail** button. Then click on the **Clear runtime prepare cache** button. If you want to add a picture, run a recipe with runtime prepare commands. (I forgot which it was) 
Image

3. **Update Projects & services page**
* [ ] delete "TODO max amount of projects"
* [ ] update this sentence "It is created when the first user service is deployed and is charged a fixed monthly cost." it is now created when you create the project itself, it doesn't wait for the first service anymore

### PR DESCRIPTION
**Pull request by notruri**
#### What Type of Change is this?

- [ ] New Page
- [x] Minor Fix
- [ ] Minor Improvement
- [ ] Major Improvement

#### Description (required)

Minor fixes

#### Related issues & labels (optional)

- /closes #95

### CODE REVIEW

**Comment by nermalcat69** [comment]
Thanks for the pr! @RuriYS 😄

File: apps/docs/content/dotnet/how-to/build-pipeline.mdx [hunk 1]
```diff
@@ -561,9 +561,9 @@ Some packages or tools can take a long time to install. Therefore, Zerops caches
 1. Content of the [build.addToRunPrepare](#copy-folders-or-files-from-your-build-container) and `run.prepareCommands` attributes didn't change from the previous deploy
 2. The custom runtime cache wasn't invalidated in the Zerops GUI.
 
-To invalidate the Zerops custom runtime cache go to `yyy`
+To invalidate the Zerops runtime cache go to your service detail in Zerops GUI, choose Service dashboard & runtime containers from the left menu and click on the Open pipeline detail button. Then click on the Clear runtime prepare cache button. If you want to add a picture, run a recipe with [run.prepareCommands](/references/zeropsyml#preparecommands-optional-1).
```
Comment (vankovap) [inline apps/docs/content/dotnet/how-to/build-pipeline.mdx]: I am sorry, this is my bad. I missed the closing quotes, it should have look like this "To invalidate the Zerops runtime cache go to your service detail in Zerops GUI, choose **Service dashboard & runtime containers** from the left menu and click on the **Open pipeline detail** button. Then click on the **Clear runtime prepare cache** button." <- quotes here!

And this was only for you -> If you want to add a picture, run a recipe with runtime prepare commands - that's where you get the picture of the button, if you decide it would be good to have it there.

I'll do better next time.

File: apps/docs/content/features/infrastructure.mdx [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
-Zerops structure has three levels - _projects_, _services_ and _containers_.
+Zerops structure has three levels - **projects**, **services** and **containers**.
 
 ## Project
 
-Project is the top-level entity, which consists of **services**. TODO max amount of projects
+Project is the top-level entity, which consists of **services**. {/* TODO max amount of projects */}
 
-To function properly, a project needs its **"core"** to be running - which contains a logger service, a statistics service, HTTP routing with SSL certificates, IP routing with built-in firewall etc. It is created when the first user service is deployed and is charged a fixed monthly cost.
+To function properly, a project needs its **"core"** to be running - which contains a logger service, a statistics service, HTTP routing with SSL certificates, IP routing with built-in firewall etc. It is now created when you create the project itself, it doesn't wait for the first service anymore.
```
Comment (vankovap) [inline apps/docs/content/features/infrastructure.mdx]: I wanted to only state that it is created right away - "It is created when you create the project itself" is fine. Since most people don't know how it worked before, it is not necessary to explain.

File: apps/docs/content/dotnet/how-to/customize-runtime.mdx [hunk 3]
```diff
@@ -16,39 +16,45 @@ import Image from '/src/components/Image';
 
 import data from '@site/static/data.json';
 
-The default .NET runtime environment contains:
+
+## Customise .NET build environment
```
Comment (vankovap) [inline apps/docs/content/dotnet/how-to/customize-runtime.mdx]: Good catch with the `build` prefix, but it was supposed to be `run.os` https://docs.zerops.io/nodejs/how-to/build-pipeline#os-1 and `run.prepareCommands` https://docs.zerops.io/nodejs/how-to/build-pipeline#os-1 instead :grimacing: Keep it as a note, it's good, but the headings can stay as they were. Also keep the paragraph in build process too.

File: apps/docs/content/dotnet/how-to/customize-runtime.mdx [hunk 4]
```diff
@@ -16,39 +16,45 @@ import Image from '/src/components/Image';
...
-    `zCLI`
-  </a>
-  , Zerops command line tool
-- ASP .NET`and`git`
+- .NET Version
+- [zCLI](/references/cli)
+- ASP .NET and Git
+
+:::note
+If you need a different OS besides Alpine, for example Ubuntu, Set the [build.os](/dotnet/how-to/build-pipeline#os) attribute to `ubuntu`.
```
Comment (vankovap) [inline apps/docs/content/dotnet/how-to/customize-runtime.mdx]: I have a slight feeling, that this sounds like we support Alpine, Ubuntu and possibly another OS too, which we don't.

**Review by vankovap — COMMENTED** [review]
Sorry, it's mostly bad instructions, I'll do better next time

File: apps/docs/content/dotnet/how-to/build-process.mdx [hunk 5]
```diff
@@ -32,20 +32,25 @@ The build container is automatically deleted after the build has finished or fai
...
 - <span>{data.alpineCurrentVersion}</span>
-- selected version of .NET defined in `zerops.yml` [build.base](/dotnet/how-to/build-pipeline#base) parameter
-- [zCLI](/references/cli), Zerops command line tool
-- ASP .NET`and`git`
+- .NET Version
+- [zCLI](/references/cli)
+- ASP .NET and Git
+
+:::note
+To use Ubuntu instead of the default Alpine, set the [run.os](/references/zeropsyml#os-optional-1) attribute.
```
Comment (vankovap) [inline apps/docs/content/dotnet/how-to/build-process.mdx]: this was correct - `os` and `prepareCommands` are in both `build` and `run` section, so in `build-process.mdx` pages there should be `build.os` and `build.prepareCommands` and in `customize-runtime.mdx` pages there should be `run.os` and `run.prepareCommands` pages

File: apps/docs/content/dotnet/how-to/customize-runtime.mdx [hunk 6]
```diff
@@ -16,39 +16,45 @@ import Image from '/src/components/Image';
 
 import data from '@site/static/data.json';
 
+
+## Customise .NET Runtime environment
```
Comment (vankovap) [inline apps/docs/content/dotnet/how-to/customize-runtime.mdx]: I would delete this one, the main title is enough.

image

File: apps/docs/content/dotnet/how-to/customize-runtime.mdx [hunk 7]
```diff
@@ -16,39 +16,45 @@ import Image from '/src/components/Image';
...
 
-1. Content of the <a href="/dotnet/how-to/build-pipeline#copy-folders-or-files-from-your-build-container">`build.addToRunPrepare`</a> and <a href="/dotnet/how-to/build-pipeline#preparecommands-1">`run.prepareCommands`</a> attributes didn't change from the previous deploy
+1. Content of the <a href="/dotnet/how-to/build-pipeline#copy-folders-or-files-from-your-build-container">build.addToRunPrepare</a> and <a href="/dotnet/how-to/build-pipeline#preparecommands-1">run.prepareCommands</a> attributes didn't change from the previous deploy
 2. The custom runtime cache wasn't invalidated in the Zerops GUI.
 
-To invalidate the custom runtime cache go to `yyy`
+To invalidate the Zerops runtime cache go to your service detail in Zerops GUI, choose **Service dashboard & runtime containers** from the left menu and click on the **Open pipeline detail** button. Then click on the **Clear runtime prepare cache** button.
```
Comment (vankovap) [inline apps/docs/content/dotnet/how-to/customize-runtime.mdx:54]: what do you think about this layout?

image

File: apps/docs/content/go/how-to/customize-runtime.mdx [hunk 8]
```diff
@@ -17,20 +17,17 @@ import data from '@site/static/data.json';
...
 - <span>{data.alpineCurrentVersion}</span>
-- selected version of Go selected when the runtime service was created.
-- <a href="/references/cli">
-    `zCLI`
-  </a>
-  , Zerops command line tool
-- `git` and `wget`
+- Selected version of Go when the runtime service was created.
+- [zCLI](/references/cli)
+- Git
```
Comment (vankovap) [inline apps/docs/content/go/how-to/customize-runtime.mdx:23]: the note about ubuntu is missing here

File: apps/docs/content/java/how-to/build-process.mdx [hunk 9]
```diff
@@ -32,21 +32,10 @@ The build container is automatically deleted after the build has finished or fai
 
 When you know that the running build is not correct and you want to cancel it, you can do it in Zerops GUI. Go to the service detail, select **Service dashboard & runtime containers** from the left menu and click on the **Open pipeline detail** button. Then click on the **Cancel build** button.
 
-**// TODO screenshot of a running build pipeline**
+{/* ** TODO screenshot of a running build pipeline** */}
 
 The build cancellation is available before the build pipeline is finished. When the build is finished, the deployment cannot be cancelled.
 
-## Customise Java build environment
```
Comment (vankovap) [inline apps/docs/content/java/how-to/build-process.mdx:39]: why is this deleted?

**Review by vankovap — CHANGES_REQUESTED** [review]
Can you go through the folders again and make sure there are the same changes in all pages of the same topic, e.g. the changes in all `build-pipeline.mdx` files are the same (of course if there are technology-specific details, keep them there)

**Comment by nermalcat69** [comment]
LGTM! 📝  🎉
